.github/workflows/ci.yml

```
@@ -141,7 +141,7 @@ jobs:
       - run: |
           cargo test --package forge --features scarb_2_8_3 e2e::features
       - run: |
-          cargo test --package scarb-api --features scarb_2_8_3 get_starknet_artifacts_path
+          cargo test --package scarb-api --features scarb_2_8_3
```

```suggestion
          cargo test --package scarb-api --features scarb_2_8_3 get_starknet_artifacts_path
```

Actually let's restore that and add a separate step below it

```yml
run: |
    cargo test --package scarb-api --features scarb_2_8_3 test_load_contracts_artifacts